src/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+add_executable(timg)
+target_sources(timg PRIVATE
+  image-display.cc
+  image-source.cc
+  renderer.cc
+  terminal-canvas.cc
+  timg.cc
+  display-options.h
+  image-source.h
+  terminal-canvas.h
+  timg-time.h
+  image-display.h
+  renderer.h
+  thread-pool.h
+  video-display.h
```

Comment:
These looks somewhat in random order, which makes it hard to add things later and discover duplicates.
I suggest to sort them (As exception maybe place `timg.cc` at the top still to indicate that this contains `main()`).

Reply:
yeah, I've just used ls' output… sorted now with timg.cc on top.
I've moved video-display.h as well, it should be in guarded by the if(WITH_VIDEO_DECODING)…